src/routers/index.js

```diff
@@ -58,6 +60,10 @@ router.get('/auth/google/callback', passport.authenticate('google', { failureRed
 
 router
   .post('/signup', checkBody, createUser);
+router.post('/login', checkFields, login);
+router.post('/followers/:followerId/follow', Authenticator.verifyToken, followUser);
+router.delete('/followers/:followerId/unfollow', Authenticator.verifyToken, unfollowUser);
```

Comment:
@Raymond-Osy if you implement as toggle functionality you don't need  unfollow endpoint

Reply:
Yes, thanks for the idea...it makes sense. Implemented.